nixos/modules/security/default.nix

```diff
@@ -0,0 +1,21 @@
+{ config, lib, ... }:
+let
+  cfg = config.security;
+in
+{
+  options = {
+    security.lsm = lib.mkOption {
+      type = lib.types.listOf lib.types.str;
+      default = [ ];
+      description = ''
+        A list of the LSMs to initialize.
```

Comment:
I think it would be nice to note that this option is *ordering sensitive*. I know that the list implies this, but still worth nothing. I've had way too many ordering-related footgun incidents while setting up LSMs.

Reply:
Done